themes/default/css/wide.css

```diff
@@ -1616,7 +1616,7 @@ input[type="text"], input[type="textarea"], textarea,  input[type="password"] {
 }
 .ui-tabs-loading em {
     padding: 0 0 0 20px;
-    background: url(loading.gif) no-repeat 0 50%;
+    background: url(images/loading.gif) no-repeat 0 50%;
```

Comment:
We should replace these icons with Font-Awesome icons (using caNavIcon()?)

Reply:
I will remove this commit, it shouldn't be in this PR.